priv/apns/dev_cert.pem

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I believe openssl is required in order to provide HTTPs from Elixir node. This probably means that openssl will have to be installed anyway. My biggest concern is with such files included in the repo, users may start using them without regenerating. For such a simple project which most probably we'll be used internally fresh certs and keys may not be mission critical. It is still sth which lights a warning in my brain, though.